tests/execution/graphql-conformance-http-007.md

```diff
@@ -0,0 +1,84 @@
+---
+skip: true
+---
+
+# Test named fragments. Skipped because Tailcall does not construct correctly the interface types based on \_\_typename field.
```

Comment:
I think it's not working because we not using our discriminator https://github.com/tailcallhq/tailcall/blob/f464794bd32abe3224eb96e1de09d6fccd766148/src/core/blueprint/union_resolver.rs#L34 for interfaces and only for unions. The upstreams in most cases won't provide `__typename` info for us anyway and we need to figure out the type on our side, that's the discrininator job